### PR TITLE
Test the return type of DriverManager::getAvailableDrivers()

### PR DESCRIPTION
**Pull request by MidnightDesign**
|      Q       |   A
|------------- | -----------
| Type         | improvement
| Fixed issues | 

#### Summary

Follow-up to https://github.com/doctrine/dbal/pull/5853#issuecomment-1369625978

### CODE REVIEW

**Review by derrabus — APPROVED** [review]

File: phpstan.neon.dist [hunk 1]
```diff
@@ -3,6 +3,7 @@ parameters:
     phpVersion: 80200
     paths:
         - src
+        - static-analysis
```
Comment (greg0ire) [inline phpstan.neon.dist:6]: Oh wow woopsie

Comment (greg0ire) [reply phpstan.neon.dist:6]: I guess this particular change should target 3.5.x?

Comment (derrabus) [reply phpstan.neon.dist:6]: > Oh wow woopsie

Yeah, the one test was only run with Psalm previously. I don't think that this is a big deal tbh.

> I guess this particular change should target 3.5.x?

I don't think that's necessary.

Comment (greg0ire) [reply phpstan.neon.dist:6]: Ah indeed, if it's running with Psalm, it's not a bug then, we can view this as an improvement.

**Comment by greg0ire** [comment]
Thanks @MidnightDesign !

**Pull request merged**